client/src/pages/tasks/Show.js

```diff
@@ -293,6 +293,29 @@ const TaskShow = ({ pageDispatchers }) => {
                       }
                     />
                   )}
+                  {task.descendantTasks && task.descendantTasks.length > 0 && (
+                    <TaskParentTask
```

Comment:
I would define a separate component `TaskChildrenTasks` for clarity instead of reusing `TaskParentTask`.